File: src/Logs/Test/Exemple.js
const { Logs } = require("rpa-logs");

globalThis.Log = new Logs();

Log.setup({
    pathfile: 'Logs',
    formatfilename: 'Log %j-%M-%a.log',
    prefixlog: '[%h:%m %j/%M/%a][%type] '
});

Log.debug("Hello World !");
Log.info("Hello World !");
Log.warning("Hello World !");
Log.error("Hello World !");
Log.critical("Hello World !");

File: src/Logs/Test/Exemple.py
import main

Log = main.Logs()

# The %H, %M and %m variables are inverted compared to node js
Log.setup({
    'pathfile': 'Logs',
    'formatfilename': 'Log %j-%m-%a.log',
    'prefixlog': '[%H:%M %j/%m/%a][%type] '
})

Log.debug("Hello world !")
Log.info("Hello world !")
Log.warning("Hello world !")
Log.error("Hello world !")
Log.critical("Hello world !")